api/lib/domain/services/certification-candidates-ods-service.js

```diff
@@ -131,3 +138,25 @@ function _handleVersionError() {
 function _handleParsingError() {
   throw new CertificationCandidatesImportError({ code: 'INVALID_DOCUMENT', message: 'Le document est invalide.' });
 }
+
+async function _buildComplementaryCertificationsFromFile({
+  cleaNumerique,
+  pixPlusDroit,
+  complementaryCertificationRepository,
```

Comment:
C'est plus un fetch ou un retrieve ou un get qu'un build ? Et j'ai l'impression que c'est plus from db que from file ?

Reply:
Je propose for line ?

Reply:
J'ai mis ForLine